http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -99,3 +105,10 @@ docker run -d --restart always --network host --name welder \
 --env STAGING_BUCKET=$WELDER_STAGING_BUCKET \
 --env SHOULD_BACKGROUND_SYNC="false" \
 $WELDER_WELDER_DOCKER_IMAGE
+
+echo "{ 'env': { 'WORKSPACE_ID': '${WORKSPACE_ID}', 'WORKSPACE_STORAGE_CONTAINER_ID': '${WORKSPACE_STORAGE_CONTAINER_ID}', 'WORKSPACE_NAME': '${WORKSPACE_NAME}', 'WORKSPACE_STORAGE_CONTAINER_URL': '${WORKSPACE_STORAGE_CONTAINER_URL}' }}" | jq . >> wsenv.json
```

Comment:
I think the quotes are a little messed up here. Json keys need double quotes instead of single. Maybe use `jq` to generate the json too? This seems to work:
```
jq --null-input --arg workspace_id $WORKSPACE_ID --arg workspace_storage_container_id $WORKSPACE_STORAGE_CONTAINER_ID --arg workspace_name $WORKSPACE_NAME --arg workspace_storage_container_url $WORKSPACE_STORAGE_CONTAINER_URL '{ "env": { "WORKSPACE_ID": $workspace_id, "WORKSPACE_STORAGE_CONTAINER_ID": $workspace_storage_container_id, "WORKSPACE_NAME": $workspace_name, "WORKSPACE_STORAGE_CONTAINER_URL": $workspace_storage_container_url }}'  > wsenv.json
```